node/src/GlideClient.ts

```diff
@@ -359,15 +373,18 @@ export class GlideClient extends BaseClient {
      */
     public async configGet(
         parameters: string[],
-    ): Promise<Record<string, string>> {
-        return this.createWritePromise(createConfigGet(parameters));
+        decoder?: Decoder,
+    ): Promise<Record<string, GlideString>> {
+        return this.createWritePromise(createConfigGet(parameters), {
+            decoder: decoder,
+        });
     }
 
     /**
-     * Set configuration parameters to the specified values.
+     * Sets configuration parameters to the specified values.
      *
      * @see {@link  https://valkey.io/commands/config-set/|valkey.io} for details.
-     * @param parameters - A List of keyValuePairs consisting of configuration parameters and their respective values to set.
+     * @param parameters - A map consisting of configuration parameters and their respective values to set.
```

Comment:
Prior to Version 7.0.0, we can only send one parameter.  
Please raise a bug to document and test this.

Reply:
https://github.com/valkey-io/valkey-glide/issues/2192